app/controllers/stories_controller.rb

```diff
@@ -38,6 +38,12 @@ def warm_comments
 
   private
 
+  def assign_hero_html
+    return if SiteConfig.campaign_hero_html_variant_name.blank?
+
+    @hero_html = HtmlVariant.relevant.select(:html).find_by(group: "campaign", name: SiteConfig.campaign_hero_html_variant_name)&.html
```

Comment:
This is also slightly to long for my taste. I'm not super particular about line length, i.e. it doesn't have to be < 80, but > 130 is certainly too long for my taste.